public/js/listeners.js

```diff
@@ -1,425 +0,0 @@
-// Most Listeners are stored here
-
-
-function startListeners() {
-    window.addEventListener('keyup', function(e) {
-        scheduleUpdate(1);
-        simulationArea.shiftDown = e.shiftKey;
-        if (e.keyCode == 16) {
-            simulationArea.shiftDown = false;
-        }
-        if (e.key == "Meta" || e.key == "Control") {
-            simulationArea.controlDown = false;
-        }
-    });
-    document.getElementById("simulationArea").addEventListener('mousedown', function(e) {
-
-        $("input").blur();
-
-        errorDetected = false;
-        updateSimulation = true;
-        updatePosition = true;
-        updateCanvas = true;
-
-        simulationArea.lastSelected = undefined;
```

Comment:
Why?

Reply:
@tachyons 
i added this files by mistake  , the i deleted it from my PR
- the only changes are at styling 
no thing at .js files are included so i deleted it after  realizing i added them

Reply:
@tachyons  
Please Help me if you know a way to remove some commits from the PR

Reply:
i will google it tomorrow and fix it :)